src/assets/fonts.css

```diff
@@ -1,74 +1,49 @@
 @font-face {
-  font-family: 'JetBrains Mono';
-  font-style: normal;
-  font-weight: 400;
-  font-display: swap;
-  src: local('JetBrainsMono-Regular'), local('JetBrains Mono Regular'),
-    url('~assets/fonts/JetBrainsMono-Regular.woff2') format('woff2'),
-    url('~assets/fonts/JetBrainsMono-Regular.woff') format('woff'),
-    url('~assets/fonts/JetBrainsMono-Regular.otf') format('opentype');
-}
-@font-face {
-  font-family: 'JetBrains Mono';
-  font-style: normal;
-  font-weight: 600;
-  font-display: swap;
-  src: local('JetBrainsMono-Bold'), local('JetBrains Mono Bold'),
-    url('~assets/fonts/JetBrainsMono-Bold.woff2') format('woff2'),
-    url('~assets/fonts/JetBrainsMono-Bold.woff') format('woff'),
-    url('~assets/fonts/JetBrainsMono-Bold.otf') format('opentype');
-}
-@font-face {
-  font-family: 'Source Sans Pro';
+  font-family: 'Inter';
   font-style: normal;
-  font-weight: 400;
+  font-weight: 500;
```

Comment:
@panchovm this looks too bold to me. I thought 400 is usually the baseline for body text?

Reply:
I noticed that we have 400 and 500 for `.has-text-weight-medium` and `.has-text-weight-normal` classes. We should have different weights.

Reply:
Based on [this reference from MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping), normal should be 400 and medium should be 500.

Reply:
I think I missed the point. Let me start again: I agree with `400` as normal for default body text, and `500` as medium for `.has-text-weight-medium` text. Are you agree with that too?